custom_components/thermal_comfort/config_flow.py

```diff
@@ -56,12 +95,12 @@ def build_schema(
             vol.Required(
                 CONF_TEMPERATURE_SENSOR,
                 default=get_value(config_entry, CONF_TEMPERATURE_SENSOR),
-            ): str,
+            ): vol.In(t_sensors),
```

Comment:
We should add a default selection e.g. Element 0 so we avoid error messages if a user tries to continue without selection.